README.rst

```diff
@@ -16,11 +16,11 @@ pre-processing implementations, resulting in challenges such as
 portability of training and inference workflows, and code
 maintainability.
 
-NVIDIA Data Loading Library (DALI) is a collection of highly optimized
+NVIDIA (R) Data Loading Library (DALI) is a collection of highly optimized
 building blocks, and an execution engine, to accelerate the
 pre-processing of the input data for deep learning applications. DALI
-provides both the performance and the flexibility for accelerating
-different data pipelines as a single library. This single library can
+provides both the performance and the flexibility to accelerate
+different data pipelines as a one library. This library can
```

Comment:
```suggestion
different data pipelines as one library. This library can
```
Either "as one" or "as a single". "as a one" is wrong.

Reply:
Done